code/clue.js

```diff
@@ -1,59 +1,158 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// OBJECTS FOR SUSPECTS
 
+//OBJECTS 
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
   color: 'green',
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur' 
 }
-
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
-
-const rope = {
-  name: 'rope',
-  weight: 10
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'pink',
+  description: 'He wears sunglasses',
+  age: 37,
+  image: 'assets/plum.png',
+  occupation: 'Project Manager' 
+}
+const missScarlet = {
```

Comment:
It is good that you use indentation here, makes your code more readable and clear!